eng/versioning/external_dependencies.txt

```diff
@@ -119,66 +99,12 @@ testdep_net.bytebuddy:byte-buddy;1.15.5
 testdep_net.bytebuddy:byte-buddy-agent;1.15.5
 
 ## Spring boot dependency versions
-org.springframework.boot:spring-boot-dependencies;2.7.18
```

Comment:
@Netyyyy, will our script regenerate and append these entries here?

Reply:
No, we won't update and use these dependencies for Spring Boot2